### PR TITLE
AVX optimizations for Exposure kernel - HOST

### PR DESCRIPTION
**Pull request by sampath1117**
*Added for U8 , F32 , I8 , F16 variants
*added for PKD3,PLN3,PLN1 with toggle

### CODE REVIEW

**Review by r-abishek — CHANGES_REQUESTED** [review]
merge conflicts

**Comment by sampath1117** [comment]
Resolved the merge conflicts @r-abishek

File: src/modules/rppt_tensor_color_augmentations.cpp [hunk 1]
```diff
@@ -770,3 +770,65 @@ RppStatus rppt_color_cast_host(RppPtr_t srcPtr,
 
     return RPP_SUCCESS;
 }
+
+/******************** exposure ********************/
+
+RppStatus
+rppt_exposure_host(RppPtr_t srcPtr,
```
Comment (r-abishek) [inline src/modules/rppt_tensor_color_augmentations.cpp:777]: Format the rppt_ right next to RppStatus on previous line

File: src/modules/cpu/kernel/exposure.hpp [hunk 2]
```diff
@@ -0,0 +1,775 @@
...
+
+        Rpp8u *srcPtrChannel, *dstPtrChannel;
+        srcPtrChannel = srcPtrImage + (roi.xywhROI.xy.y * srcDescPtr->strides.hStride) + (roi.xywhROI.xy.x * layoutParams.bufferMultiplier);
+        dstPtrChannel = dstPtrImage;
+
+        Rpp32u alignedLength = (bufferLength / 48) * 48;
+        Rpp32u vectorIncrement = 48;
+        Rpp32u vectorIncrementPerChannel = 16;
+
+
```
Comment (r-abishek) [inline src/modules/cpu/kernel/exposure.hpp:41]: Remove extra blank line

**Review by r-abishek — CHANGES_REQUESTED** [review]
A few minor formatting changes

**Comment by sampath1117** [comment]
Resolved the review comments @r-abishek

**Review by r-abishek — APPROVED** [review]

**Pull request merged**